cps/main.c

```diff
@@ -1088,9 +1092,19 @@ run_cps(struct net_config *net_conf, struct gk_config *gk_conf,
 		goto stage3;
 	}
 
+	if (gk_conf != NULL) {
+		cps_conf->bgp_max_pkt_burst =
+			RTE_MAX(gk_conf->front_max_pkt_burst,
+				gk_conf->back_max_pkt_burst);
+	}
+
+	if (gt_conf != NULL)
+		cps_conf->bgp_max_pkt_burst = gt_conf->gt_max_pkt_burst;
+
+	ele_size += sizeof(struct rte_mbuf *) * cps_conf->bgp_max_pkt_burst;
```

Comment:
It's better to rename field `bgp_max_pkt_burst` to `mailbox_max_pkt_burst` because we may eventually need this information for another protocol.

Reply:
Moreover, like in the LLS case, we need to take a max of `sizeof(struct cps_request)` and all variable requests. See LLS comment regard to his issue for an example.